Shared/Tracking/Trackers/shikimori/ShikimoriTracker.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ShikimoriTracker.swift
+//  Aidoku
+//
+//  Created by Vova Lapskiy on 02.11.2024.
+//
+
+import Foundation
+
+#if os(OSX)
+    import AppKit
+#else
+    import UIKit
+#endif
+
+class ShikimoriTracker: OAuthTracker {
+    var api = ShikimoriApi()
```

Comment:
not mutable so should be let (also applies to `scoreType`)
```suggestion
    let api = ShikimoriApi()
```